everviz/plugins/_crossplot_plugin/crossplot_plugin.py

```diff
@@ -0,0 +1,82 @@
+from uuid import uuid4
+import numpy as np
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import plotly.graph_objs as go
+from webviz_config import WebvizPluginABC
+
+from everviz.plugins._crossplot_plugin._dash_components._crossplot_layout import (
+    _get_layout,
+)
+from everviz.plugins._crossplot_plugin._dash_components._crossplot_callback import (
+    _crossplot_callback,
+)
+from everviz.data.load_csv._get_data import get_data
+from everviz.plugins._crossplot_plugin.crossplot_control_group_plugin import _get_trace
+
+
+class CrossplotPlugin(WebvizPluginABC):
+    def __init__(self, app, data_path, title="Crossplot"):
+        super().__init__()
+        self.title = title
+        self.graph_id = f"graph-{uuid4()}"
+        self.dropdown_x_id = f"dropdown-{uuid4()}"
+        self.dropdown_y_id = f"dropdown-{uuid4()}"
+        self.radio_items_x_id = f"radio-{uuid4()}"
+        self.radio_items_y_id = f"radio-{uuid4()}"
+
+        self.data_path = data_path
+        self.set_callbacks(app)
+
+    def add_webvizstore(self):
+        return [(get_data, [{"data_path": self.data_path}])]
+
+    @property
+    def layout(self):
+        df = get_data(self.data_path)
+        dropdown_options = df.columns.unique()
+        basic_layout = _get_layout(
+            self.title,
+            dropdown_options,
+            self.graph_id,
+            self.dropdown_x_id,
+            self.dropdown_y_id,
+            self.radio_items_x_id,
+            self.radio_items_y_id,
+        )
+        return html.Div(basic_layout, style={"horizontal-align": "middle"})
+
+    def set_callbacks(self, app):
+        @app.callback(
+            Output(self.graph_id, "figure"),
+            [
+                Input(self.dropdown_x_id, "value"),
+                Input(self.dropdown_y_id, "value"),
+                Input(self.radio_items_x_id, "value"),
+                Input(self.radio_items_y_id, "value"),
+            ],
+        )
+        def update_graph(xaxis_column_name, yaxis_column_name, xaxis_type, yaxis_type):
+            df = get_data(self.data_path)
+            data = [
+                _get_trace(
+                    df, xaxis_column_name, yaxis_column_name, xaxis_type, yaxis_type
+                )
+            ]
+            return _crossplot_callback(
+                data, xaxis_column_name, yaxis_column_name, xaxis_type, yaxis_type
+            )
+
+
+def _get_trace(df, xaxis_column_name, yaxis_column_name, xaxis_type, yaxis_type):
+    try:
+        x_data = df[xaxis_column_name]
+        y_data = df[yaxis_column_name]
+    except KeyError:
+        return go.Scatter()
+    return go.Scatter(
+        y=y_data if yaxis_type != "Cumulative" else np.cumsum(y_data),
+        x=x_data if xaxis_type != "Cumulative" else np.cumsum(x_data),
+        mode="markers",
+        marker={"size": 10},
```

Comment:
There is styling scattered around the layouts, we should have it more centralized. Would be good to have it separated in a css file instead, unsure if it would have any problems with the `--theme` in webviz-config, but I don't think so.

Reply:
Lets discuss this on Monday.